src/commit_list.c

```diff
@@ -9,7 +9,10 @@
 
 #include "revwalk.h"
 #include "pool.h"
+#include "repository.h"
 #include "odb.h"
+#include "commit.h"
+#include "signature.h"
```

Comment:
I couldn't really see why repository.h and signature.h are required now. At least that's not obvious from the diff.

Reply:
Leftover from a previous attempt, where I tried to make that parser use already existing functions, before I noticed the "main" parse code. Will remove if that's unwarranted (I don't think it is).